backend/src/main/java/pl/edu/agh/io/backend/controllers/VisualizerController.java

```diff
@@ -0,0 +1,26 @@
+package pl.edu.agh.io.backend.controllers;
+
+import org.springframework.web.bind.annotation.*;
+import pl.edu.agh.io.backend.entities.response.ResponseData;
+import pl.edu.agh.io.backend.services.VisualizerService;
+import pl.edu.agh.io.backend.entities.config.JsonConfig;
+
+@RestController
+public class VisualizerController {
+    private final VisualizerService visualizerService;
+
+    private VisualizerController(VisualizerService visualizerService) {
```

Comment:
Po co prywatny konstruktor?

Reply:
> 

On jest tylko po to, żeby zrobić dependency injection